dex-it/src/test/scala/com/wavesplatform/it/sync/orderV3/FixedModeMatcherTestSuite.scala

```diff
@@ -0,0 +1,253 @@
+package com.wavesplatform.it.sync.orderV3
```

Comment:
Partially filled orders, cancelled orders, rounding of small fees, asset pairs with different decimals cound
See below in "dex-it/src/test/scala/com/wavesplatform/it/sync/orderV3/PercentModeMatcherTestSuite.scala"